### ISSUE TITLE
Implement the paper PPO with SB3

### ISSUE THREAD

**Issue by TeoIlie** [TeoIlie/F1TENTH_Gym#23]
Implement the following features for my DRL model:
1. PPO
2. Parallelized with 400 simulated instances running concurrently
3. Each rollout consists of 1024 steps with a time step of 0.05s
4. Training with batch size of 1024
5. Sum of discounted reward is calculated with discount factor gamma=0.99
6. Learning rate decays from 1 x 10^-3 to 1 x 10^-4 over the course of training
7. 120 million time steps
8. Actor and critic networks implemented as MLPs
9. Actor has 2 hidden layers of 256 neurons
10. Critic has two hidden layers of 512 neurons
11. Both actor, critic networks use Leaky Rectified Linear Unit activation function with negative slop 0.2

Some other nice-to-haves:
1. Integration with **wandb**
2. Integration with **tensorboard** to see the reward of training live, and visually see when it begins to converge.

**Comment by TeoIlie** [comment]
The useful subset of the requirements was implemented - will add the full feature set in future as necessary. Closing for now.
